frontend-react/src/components/Admin/OrgsTable.tsx

```diff
@@ -67,13 +66,19 @@ export function OrgsTable() {
                         eachOrg.jurisdiction,
                         eachOrg.stateCode,
                         eachOrg.countyName,
-                        new Date(eachOrg.meta.createdAt).toDateString(),
+                        // eachOrg.createdAt
```

Comment:
Left in for later use or forgot to remove?

Reply:
I left it in for later, but happy to remove. Not sure how likely we are to ever get or need this